### ISSUE TITLE
NvimTreeVertSplit highlight group is gone?

### ISSUE THREAD

**Issue by Tummetott** [kyazdani42/nvim-tree.lua#1142]
### Description

I used to set a custom color for the `NvimTreeVertSplit` highlight group which is different from the `VertSplit` group. It seems like with the latest update, this highlight group is gone? Why is that the case?

### Neovim version

```markdown
NVIM v0.7.0-dev+1406-g8c25dbff4
Build type: Release
LuaJIT 2.1.0-beta3
```

### Operating system and version

macOS 10.15.7

### nvim-tree version

618ea25

### Steps to reproduce

```lua
vim.cmd('hi NvimTreeVertSplit guibg=#ff0000 guifg=#00FF00')
```

### Expected behavior

The vertical split line is colored in red/green

### Actual behavior

The vertical split line has the colors from the `VertSplit` highlight group

### Minimal config

```Lua
vim.cmd [[set runtimepath=$VIMRUNTIME]]
vim.cmd [[set packpath=/tmp/nvt-min/site]]
local package_root = "/tmp/nvt-min/site/pack"
local install_path = package_root .. "/packer/start/packer.nvim"
local function load_plugins()
  require("packer").startup {
    {
      "wbthomason/packer.nvim",
      "kyazdani42/nvim-tree.lua",
      "kyazdani42/nvim-web-devicons",
    },
    config = {
      package_root = package_root,
      compile_path = install_path .. "/plugin/packer_compiled.lua",
      display = { non_interactive = true },
    },
  }
end
if vim.fn.isdirectory(install_path) == 0 then
  print "Installing nvim-tree and dependencies."
  vim.fn.system { "git", "clone", "--depth=1", "https://github.com/wbthomason/packer.nvim", install_path }
end
load_plugins()
require("packer").sync()
vim.cmd [[autocmd User PackerComplete ++once echo "Ready!" | lua setup()]]
vim.opt.termguicolors = true
vim.opt.cursorline = true

_G.setup = function()
  require("nvim-tree").setup {}
end
```

**Comment by alex-courtis** [comment]
I can't reproduce this one:
```
nvim -nu /tmp/nvt-min.lua
:NvimTreeOpen
:hi NvimTreeVertSplit guibg=#ff0000 guifg=#00FF00
```

![2022-04-09-153323_974x789_scrot](https://user-images.githubusercontent.com/1505378/162558040-cd9ffa7e-2f8a-4457-9ed8-103722d0a80b.png)

**Comment by kyazdani42** [comment]
might be a colorscheme overriding this ? try `:hi NvimTreeVertSplit` after neovim is loaded, it shouldn't be `cleared`

**Comment by HumblePresent** [comment]
I am experiencing the same issue. The output of `:set winhl?` in the nvim-tree window is
```
winhighlight=EndOfBuffer:NvimTreeEndOfBuffer,Normal:NvimTreeNormal,CursorLine:NvimTreeCursorLine,VertSplit:NvimTreeVertSplit,StatusLine:NvimTreeStatusLine,StatusLineNC:NvimTreeStatuslineNC,SignColumn:NvimTreeSignColumn,NormalNC:NvimTreeNormalNC
```
The output of `:hi NvimTreeVertSplit` matches the custom highlight I have set.
```
NvimTreeVertSplit xxx guifg=#282828 guibg=#f2594b
```
When I change the `VertSplit` highlight on the command line the nvim-tree vertical split changes to match so it's definitely using the `VertSplit` highlight group and seems to be ignoring the `winhl` option.

**Comment by HumblePresent** [comment]
I downgraded from Neovim nightly to stable 0.6.1 and am no longer experiencing the issue, so it appears to be an upstream issue.

**Comment by alex-courtis** [comment]
> I downgraded from Neovim nightly to stable 0.6.1 and am no longer experiencing the issue, so it appears to be an upstream issue.

Good catch.

If you can replicate under `nvim --clean` without nvim-tree.lua you can raise a [neovim issue](https://github.com/neovim/neovim/issues/new/choose).

**Comment by alex-courtis** [comment]
Closing for now. Please reopen if this is a problem when neovim 0.7 is released.

**Comment by siduck** [comment]
> hi NvimTreeVertSplit guibg=#ff0000 guifg=#00FF00

I face this issue with 0.7.0!

**Comment by siduck** [comment]
is this highlight group replaced with WinSeparator?

**Comment by Tummetott** [comment]
> > hi NvimTreeVertSplit guibg=#ff0000 guifg=#00FF00
> 
> I face this issue with 0.7.0!

This is only a problem since nvim 0.7

**Comment by siduck** [comment]
> > > hi NvimTreeVertSplit guibg=#ff0000 guifg=#00FF00
> > 
> > 
> > I face this issue with 0.7.0!
> 
> This is only a problem since nvim 0.7

found any fix :(

**Comment by Tummetott** [comment]
Nope unfortunately not. Apparently the problem is upstream but i don't understand what the problem is.

**Comment by siduck** [comment]
> Nope unfortunately not. Apparently the problem is upstream but i don't understand what the problem is.

it would be nice to make an issue there

**Comment by HumblePresent** [comment]
I was able to reproduce this with with `nvim --clean` on Nvim 0.7.0 so it is an upstream issue. I haven't created an issue there yet though as I've been a little short of time lately.

**Comment by siduck** [comment]
@kyazdani42 @alex-courtis 


https://github.com/neovim/neovim/issues/18325#issuecomment-1113999533

**Comment by kyazdani42** [comment]
closed in #1219
